docs/src/data/test/__snapshots__/sitemap.test.ts.snap

```diff
@@ -0,0 +1,992 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Sitemap Snapshot 1`] = `
+"<?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?>
+<urlset
+  xmlns=\\"http://www.sitemaps.org/schemas/sitemap/0.9\\"
+  xmlns:news=\\"http://www.google.com/schemas/sitemap-news/0.9\\"
+  xmlns:xhtml=\\"http://www.w3.org/1999/xhtml\\"
+  xmlns:mobile=\\"http://www.google.com/schemas/sitemap-mobile/1.0\\"
+  xmlns:image=\\"http://www.google.com/schemas/sitemap-image/1.1\\"
+  xmlns:video=\\"http://www.google.com/schemas/sitemap-video/1.1\\"
+>
+  <url>
+    <loc>http://localhost:3000/</loc>
```

Comment:
for my understanding, should this be localhost or actual url?

Reply:
Good point. For the e2e test, localhost should be fine.

But I'll need to update the PR since the snapshot has a timestamp 😂